### PR TITLE
Hey.

### PR DESCRIPTION
**Pull request by MichelKazi**
hey

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ jobs:
     working_directory: ~/hello-world
     steps:
       - checkout
+
+      # specify any bash command here prefixed with `run: `
```
Comment (DarthHater) [inline .circleci/config.yml]: Hey, can you get rid of this comment?

Comment (MichelKazi) [reply .circleci/config.yml]: Sure thing bro

File: Dockerfile.old [hunk 2]
```diff
@@ -0,0 +1,37 @@
+FROM golang:alpine as builder
```
Comment (DarthHater) [inline Dockerfile.old]: I don't really need this file, can you remove this?

**Review by DarthHater — CHANGES_REQUESTED** [review]
Left a couple comments, needs some clean up

File: .circleci/config.yml (hunk 1 above)
Comment (DarthHater) [reply .circleci/config.yml]: Did that in: 13d72a8

File: Dockerfile.old (hunk 2 above)
Comment (DarthHater) [reply Dockerfile.old]: Done in: 13d72a8

**Review by DarthHater — APPROVED** [review]
LGTM!

**Pull request merged**